src/client.h

```diff
@@ -62,6 +63,36 @@ class Client : public ClientBase {
     PongModeCare m_pongModeCare; // has msg_sendto() method and can be one of: PongModeNormal,
                                  // PongModeAlways, PongModeNever
 
+    class ClientMessageHandlerCallback {
```

Comment:
This class and other similar classes can be replaced with a single lambda function if we decide to use c++11.